front/src/pages/Home.tsx

```diff
@@ -66,6 +67,9 @@ function App() {
         ))}
       </div>
 
+    <div className="pb-16">
```

Comment:
# Two things:
1. When using a component, try to use the same name when using it as you did when creating it.
> When you do a default export, you're not *forced* to do this by the IDE, but it's a best practice.

2. This wrapper doesn't make sense to me, notably since our button component is absolutely placed on the page. I think you can remove it, no?